src/gsi/read_obs.F90

```diff
@@ -1523,20 +1523,35 @@ subroutine read_obs(ndata,mype)
                 if( trim(infile) == 'vr_vol' )then
                   call read_radar_wind_ascii(nread,npuse,nouse,infile,lunout,obstype,sis,&
                                   hgtl_full,nobs_sub1(1,i))
+                  string='READ_RADAR_WIND'
```

Comment:
Same question as before, are all these changes to read_obs appropriate for this PR, or should it be separate?

Reply:
Into a separate PR.